lib/Conversion/VerifToSV/VerifToSV.cpp

```diff
@@ -128,6 +128,126 @@ struct HasBeenResetConversion : public OpConversionPattern<HasBeenResetOp> {
   }
 };
 
+namespace VerifAssertLikeOp {
+
+static sv::EventControl verifToSVEventControl(verif::ClockEdge ce) {
+  switch (ce) {
+  case verif::ClockEdge::Pos:
+    return sv::EventControl::AtPosEdge;
+  case verif::ClockEdge::Neg:
+    return sv::EventControl::AtNegEdge;
+  case verif::ClockEdge::Both:
+    return sv::EventControl::AtEdge;
+  }
+  llvm_unreachable("Unknown event control kind");
+}
+
+// Convert the verif.assert like op that uses an enable, into an equivalent
+// sv.assert_property op that has the negated enable as its disable.
+template <typename TargetOp, typename Op, typename Adaptor>
+static LogicalResult convert(Op op, Adaptor operands,
+                             ConversionPatternRewriter &rewriter) {
+  // negate the enable if it exists
+  Value enable, wdisable;
```

Comment:
Nit: `if (auto enable = operands.getEnable())` since it's not used outside the if block? And as a result can drop the parens re:assignment.